xtable-api/src/main/java/org/apache/xtable/conversion/ExternalTable.java

```diff
@@ -38,6 +40,8 @@ class ExternalTable {
   protected final @NonNull String formatName;
   /** The path to the root of the table or the metadata directory depending on the format */
   protected final @NonNull String basePath;
+  /** The path to the data files, defaults to the metadataPath */
```

Comment:
There is no field for `metadataPath` in `ExternalTable`. I guess it was supposed to be the `basePath`

Reply:
Fixed the comment.